CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features
 
+- Add `SentryWidget` ([#1846](https://github.com/getsentry/sentry-dart/pull/1846))
+- APM for isar ([#1726](https://github.com/getsentry/sentry-dart/pull/1726))
```

Comment:
`- APM for isar ([#1726](https://github.com/getsentry/sentry-dart/pull/1726))` was released in 7.15.0.

Reply:
There was a mistake. We didn't add isar to the craft file so it was never published, I removed it from the 7.15.0 changelog